docs/Env_Setup.rst

```diff
@@ -108,6 +108,188 @@ with a minimal fedora 23 installation. It will leave any logs under the `logs`
 directory and any generated artifacts under `exported-artifacts`.
 
 
-
    .. _`official yapf repo`: https://github.com/google/yapf
    .. _`copr`: https://copr.Fedoraproject.org/coprs/fulltext/?fulltext=yapf
+
+
+Running Lago in virtualenv
+---------------------------
+This was tested on a fresh Fedora-24 Cloud image, but should work on Fedora 23
+and most likely with little package-names resolving on CentOS >7.
+
+Install dependencies::
+
+  > sudo dnf install -y redhat-rpm-config libvirt libvirt-devel \
+     libguestfs-devel libguestfs-tools qemu-img qemu-kvm \
+     libvirt-daemon git gcc libffi-devel python-devel \
+     openssl-devel bats python-magic python-libguestfs
+
+Note that ``python-magic`` and ``python-libguestfs`` are currently not
```

Comment:
This is great news, 
Few weeks ago I contacted the maintainer of the RPM package we use now (python-magic) and he replied they do not have packing for pip/PyPi. So I assume 'file-magic' is a different package. Were you able to test it? 
Also - If it doesn't have RPM packaging, we will still need to use the python-magic, as they might diverge with time(python-magic and file-magic).